src/scenes/_Router/GalleryRoute.tsx

```diff
@@ -57,6 +66,12 @@ export default function GalleryRoute({
     queryRef
   );
 
+  const { timestamp } = useTimer();
+
+  const countdownTimer = useMemo(() => {
+    return `Ends in ${timestamp}`;
+  }, [timestamp]);
```

Comment:
<no action required, just an observation!>

it feels a bit weird defining this in the `GalleryRoute.tsx` file as the banner won't be rendered on every screen (especially before the poster page is live). so we'll essentially have this `countdownTimer` running in the background for every GalleryRoute that's used.

overall i'm not worried because this will be a temporary feature, and this will not cause unnecessary re-renders. so performance will be fine.

one way to address this in the future may be to define a separate PosterBanner that encapsulates the countdown logic:
```js
// GalleryRoute.tsx
const bannerComponent = useMemo(() => {
  return isFeatureEnabled(FeatureFlag.POSTER_PAGE)
    ? <PosterBanner queryRef={query} />
    : <Banner text="" queryRef={query} />
}, [])

...

// PosterBanner.tsx
const PosterBanner = ({ queryRef }) => {
  const { timestamp } = useTimer();

  const countdownTimer = useMemo(() => {
    return `Ends in ${timestamp}`;
  }, [timestamp]);
  ...
}
```

Reply:
Agree with you robin. It felt weird let this function running in the background although we didn't used it. 

Thanks for the suggestion!

Sorry robin. I had to do it  Just abstract this to his own component...